tasks

```diff
@@ -238,12 +208,6 @@ function tf_apply_certs {
   terraform apply certs_deployment.tfplan
 }
 
-function tf_apply_db_roles {
-  tf_init_db_roles
-  terraform get # modules
-  terraform apply --parallelism=1 "db-roles.tfplan"
```

Comment:
Kris - Just as a safety check, are we certain this has nothing to do with the state lock?